actions/templates/post/index.js

```diff
@@ -125,7 +125,7 @@ async function main (params) {
     if (consoleProjectUrl) {
       const projectId = consoleProjectUrl.split('/').at(-2);
       const accessToken = await generateAccessToken(params.IMS_AUTH_CODE, params.IMS_CLIENT_ID, params.IMS_CLIENT_SECRET, params.IMS_SCOPES, logger);
-      const consoleClient = await consoleLib.init(accessToken, params.IMS_CLIENT_ID, 'stage'); // Dev console templates can only be added from stage
+      const consoleClient = await consoleLib.init(accessToken, params.IMS_CLIENT_ID, getEnv(logger));
```

Comment:
Hey @MichaelGoberling. Looks like you're changing the Console env the post API connects to. Shouldn't the post API always connect to console stage since templates can only be created from projects on stage console?

Reply:
Hmm okay I opened this PR because I was testing prod and it wasn't working (due to trying to use stage ims)

I thought we decided a few weeks back that templates could be made from any env and that stage-only would be enforced at the Crusher level, is it still necessary to enforce this at the TR level?

Reply:
> we decided a few weeks back that templates could be made from any env and that stage-only would be enforced at the Crusher level, is it still necessary to enforce this at the TR level?

Ah, okay. Yes, I remember. Agree that we don't need to enforce this at TR level. The Console API TR is calling here is not enabled on prod and will return a 404 I believe. So, we should be good here.

Reply:
> The Console API TR is calling here is not enabled on prod and will return a 404 I believe

Okay so we can't call the get install config API in prod? I think we need to change the POST template behavior then to support taking in a full template object and bypassing get install config, because right now it always tries to get install config if there's a console project URL

Reply:
> Okay so we can't call the get install config API in prod?

Yes, the get install config API is not available in prod.

> I think we need to change the POST template behavior then to support taking in a full template object and bypassing get install config, because right now it always tries to get install config if there's a console project URL

We already support taking in a full template object and bypass get install config if the console project url is not provided, right?